demo/clean-kubernetes.sh

```diff
@@ -6,9 +6,11 @@ set -aueo pipefail
 source .env
 
 for ns in "$BOOKWAREHOUSE_NAMESPACE" "$BOOKBUYER_NAMESPACE" "$BOOKSTORE_NAMESPACE" "$BOOKTHIEF_NAMESPACE" "$K8S_NAMESPACE"; do
-    kubectl delete namespace "$ns" --ignore-not-found
+    kubectl delete namespace "$ns" --ignore-not-found &
```

Comment:
Since the demo relies on cleanup to happen first before setting up the run, not waiting for this to complete will result in race conditions where a namsepace is being created while it is pending deletion.

Reply:
This is regarding a race where the namespace is marked for deletion but doesn't get deleted, seen this previously. This issue is present even without your change, wondering if it can be addressed.

Reply:
surprising k8s doesn't serialize the events.
should I infer that the `wait` for the bash commands will just then guarantee they are "marked for deletion" but not deleted?

Reply:
That makes sense.

The only change I would make is to delete multiple namespaces via a single command.